Formula/libpeas.rb

```diff
@@ -10,18 +10,42 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
```

Comment:
I think if we have explicit option text up here for `"with-python"` then we should also have explicit option text up here for `"with-python3"`

Reply:
Actually there will be no `"with-python"` option so we just need option text for python3.

Reply:
Implemented with cb02cfdf69c2b4a501f3614463c3844ffff88925.